src/js/view/view.js

```diff
@@ -307,12 +307,31 @@ define([
         this.handleColorOverrides = function() {
             var id = _model.get('id');
 
-            function addStyle(elements, attr, value) {
+            function getRgba(color, opacity) {
+                var canvas, context, data;
+
+                canvas = document.createElement('canvas');
+                canvas.height = 1;
+                canvas.width = 1;
+
+                context = canvas.getContext('2d');
+                context.fillStyle = color;
+                context.fillRect(0, 0, 1, 1);
+
+                data = context.getImageData(0, 0, 1, 1).data;
```

Comment:
This can be done with math is color is always a hex string.

If we're uncertain of the format and must use this method we should reuse the same canvas/context.